scripts/hudson/narayana.sh

```diff
@@ -358,8 +358,8 @@ function clone_as {
     rm -rf .git/rebase-apply
   else
     echo "First time checkout of WildFly"
-    git clone git://github.com/jbosstm/jboss-as.git -o jbosstm
-    [ $? -eq 0 ] || fatal "git clone git://github.com/jbosstm/jboss-as.git failed"
+    git clone https://github.com/jbosstm/jboss-as.git -o jbosstm
```

Comment:
@mayankkunwar it looks like this git handling is not in master branch: https://github.com/jbosstm/narayana/blob/master/scripts/hudson/narayana.sh#L361-L366

If we already need this change for cloning this repo, please can you raise a separate PR for that? /cc @mmusgrov

Reply:
Okay @tomjenkinson I will raise a separate PR for this

Reply:
Thank you